### PR TITLE
fix: ACNA-885 - aio app run's auto-logger shows last action log for last activation even before the command was run

### LINKED ISSUE #277: aio app run's auto-logger shows last action log for last activation even before the command was run
**Describe the bug**
The auto action logger will show the last log even if there where no new invocations after the `aio app run` command was run

**To Reproduce**
1. `aio app init`
2. `aio app run`
3. curl the action with the header`X-OW-EXTRA-LOGGING: ON`
4. wait for the expected log message
5. CTRL + C to kill the run server
6. `aio app run`
7. the old log appears

**Expected behavior**
only new logs for activation triggered after the `aio app run` command should show

### PR DESCRIPTION
**Pull request by shazron**
Closes #277 

## How Has This Been Tested?

See steps in #277 

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: src/lib/runDev.js [hunk 1]
```diff
@@ -241,16 +246,19 @@ async function runDev (args = [], config, options = {}, log = () => {}) {
 }
 
 /** @private */
-async function logListener (args) {
+async function logListener (pollArgs) {
+  const { limit, startTime } = pollArgs.logOptions
   try {
     // TODO : Is is better to just tail ?
```
Comment (moritzraho) [inline src/lib/runDev.js:252]: let's remove that old comment :)

**Review by moritzraho — APPROVED** [review]
LGTM

File: src/lib/runDev.js [hunk 2]
```diff
@@ -227,9 +226,15 @@ async function runDev (args = [], config, options = {}, log = () => {}) {
     log('press CTRL+C to terminate dev environment')
 
     if (config.app.hasBackend && fetchLogs) {
+      const pollArgs = {
+        config: devConfig,
+        logOptions: {
+          startTime: new Date().valueOf()
```
Comment (purplecabbage) [inline src/lib/runDev.js:232]: The cool kids use `Date.now()` these days

**Review by purplecabbage — APPROVED** [review]

File: src/lib/runDev.js (hunk 2 above)
Comment (shazron) [reply src/lib/runDev.js]: heh :) done

File: src/lib/runDev.js (hunk 1 above)
Comment (shazron) [reply src/lib/runDev.js]: done :)

**Pull request merged**